src/sonic-config-engine/sonic-cfggen

```diff
@@ -137,6 +137,13 @@ def ip_network(value):
         return "Invalid ip address %s" % value
     return r_v.network
 
+def load_namespace_config(asic_name):
+    if not SonicDBConfig.isInit():
+        if is_multi_asic():
+            SonicDBConfig.load_sonic_global_db_config(namespace=asic_name)
```

Comment:
>load_sonic_global_db_config [](http://example.com/codeflow?start=26&length=27)

Could you use `SonicDBConfig.initializeGlobalConfig`? yours is for backward-compatible purpose only, should be deprecated in new code.

Reply:
Changed

Reply:
@wumiaont - would you please also raise same change in the master PR version so that changes are consistent and not only in 202205? Thanks.

Reply:
> @wumiaont - would you please also raise same change in the master PR version so that changes are consistent and not only in 202205? Thanks.

@rlhui This is a port fix from master. Yes. that fix has some issue. I am going to fix this in master also after this is got into 2205.

Reply:
> > @wumiaont - would you please also raise same change in the master PR version so that changes are consistent and not only in 202205? Thanks.
> 
> @rlhui This is a port fix from master. Yes. that fix has some issue. I am going to fix this in master also after this is got into 2205.

Master fix was this one https://github.com/sonic-net/sonic-buildimage/issues/13137. Has a slight issue as checking the incorrect flag. Will correct this in master.